src/Components/Lifecycle/lifecycle.tsx

```diff
@@ -228,6 +274,7 @@ const LifecycleTab: React.FC<React.PropsWithChildren> = () => {
             <>
               <LifecycleChart lifecycleData={filteredChartData} />
               <LifecycleTable lifecycleData={filteredTableData} />
+              <AppLifecycleTable data={App}/>
```

Comment:
```suggestion
             <AppLifecycleTable data={AppLifecycleChanges}/>
```
this is what I'm using there now and makes sense to me. But doesn't work :(